packages/wallets/src/components/WalletsCarouselContent/WalletsCarouselContent.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import useEmblaCarousel from 'embla-carousel-react';
+import { useWalletAccountsList } from '@deriv/api';
+import { ProgressBar } from '../ProgressBar';
+import { WalletCard } from '../WalletCard';
+import { WalletListCardIActions } from '../WalletListCardIActions';
+import './WalletsCarouselContent.scss';
+
+const WalletsCarouselContent: React.FC = () => {
+    const [wallet_embla_ref, emblaApi] = useEmblaCarousel({ skipSnaps: true, containScroll: false });
+    const [active_index, setActiveIndex] = useState(0);
+    const { data: wallet_accounts_list } = useWalletAccountsList();
+
+    React.useEffect(() => {
+        emblaApi?.scrollTo(active_index);
+    }, [active_index, emblaApi]);
+
+    React.useEffect(() => {
+        emblaApi?.on('select', () => {
+            const scroll_snap_index = emblaApi.selectedScrollSnap();
+            setActiveIndex(scroll_snap_index);
+        });
+    }, [emblaApi]);
+    const amount_of_steps = Array.from({ length: wallet_accounts_list.length }, (_, idx) => idx + 1);
```

Comment:
I think we need to pass down list of loginid(s) instead here. So, we can do account switching in `ProgressBar` component. It can be done in this pr or my pr later, both work for me.

Reply:
That one can go once the switch account is done 😸